contribute/style-guides/documentation-style-guide.md

```diff
@@ -2,70 +2,69 @@
 
 This style guide applies to all documentation created for Grafana products.
 
-For information about how to write technical documentation, we suggest reviewing the content of the [Google Technical Writing courses](https://developers.google.com/tech-writing).
-
-The [Divio documentation system](https://documentation.divio.com/) site and the [Vue writing principles](https://v3.vuejs.org/guide/contributing/writing-guide.html#principles) are also good resources.
+For information about how to write technical documentation, refer to [Google Technical Writing courses](https://developers.google.com/tech-writing), the [Divio documentation system](https://documentation.divio.com/) site, and the [Vue writing principles](https://v3.vuejs.org/guide/contributing/writing-guide.html#principles).
```

Comment:
Since these are now combined, I suggest converting the links to an unordered list.